src/channel.ts

```diff
@@ -343,6 +356,9 @@ export class Channel<
       throw Error(`Invalid type ${typeof query} for query parameter`);
     }
 
+    if (sort) {
+      payload.sort = normalizeQuerySort(sort);
```

Comment:
There are also some type issues here because of sort being any...

I think bringing these all into one type sorts it all if that's acceptable for the usecase.
```
export type SearchOptions = {
  limit?: number;
  next?: string;
  offset?: number;
  sort?: SearchMessageSort;
};
```